doc/conf.py

```diff
@@ -240,6 +242,37 @@
 
 numpydoc_show_class_members = False
 
+# Make numpydoc to generate plots for example sections
+numpydoc_use_plots = True
+
```

Comment:
Since we're adding the plots outside the examples section, we don't need this config option either - much simpler!

```suggestion
```